qoute/style.css

```diff
@@ -0,0 +1,245 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Playfair Display', serif;
+
+}
+body{
+    background-color: #EFF2F1;
+    position: relative;
+}
+.index-page{
+    width: 375px;
+    height: 667px;
+    text-align: center;
+    margin: 0 auto;
+
+}

```

Comment:
**add the property `position : relative` to fix footer style at the mobile screen**